app/views/admin/suppliers/delivery_partners/_lead_providers_cohorts.html.erb

```diff
@@ -1,6 +1,6 @@
 <%= f.govuk_check_boxes_fieldset(
   :lead_provider_ids,
-  legend: { text: "Which lead providers and cohorts are #{@delivery_partner_form.name} working with?", size: "m" },
+  legend: { text: "Which lead providers and cohorts are #{@delivery_partner_form.name} working with?", tag: "h1", size: "l" },
```

Comment:
I don't think it's wise to have structurally important headers in a component that doesn't say it does that - `app/views/admin/suppliers/delivery_partners/edit.html.erb` will have two h1 headers because of this change.

Reply:
Good catch